wordpress-mobile/WordPress-Android
# Fix text color for Compose theme in night mode

Fixes #17085

> The implementation applies the suggestion from here: https://github.com/wordpress-mobile/WordPress-Android/pull/16934#discussion_r929884451.

To test:

0. Make sure the device is set to night mode
1. Open Jetpack app and Login if you're not already logged in
3. Tap `Me`
4. Tap `Scan Login Code`
5. In a guest Chrome session on a computer, go to www.wordpress.com and 
   - Tap `Log In`
   - Tap `Login via the mobile app`
6. Scan the QR Code with your device
7. **Expect** the title text to be colored white → making the text readable.

## Regression Notes
1. Potential unintended areas of impact
  Title text color in `day mode` on the screens after scanning the login code.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
  Manual testing & compose preview.

3. What automated tests I added (or what prevented me from doing so)
  No tests warranted by UI tweaks.


### Previews

| Before | After |
| --- | --- |
| ![d_old](https://user-images.githubusercontent.com/4588074/186404599-92cd85c0-7c35-411f-9381-d1729dba23b1.jpg) | ![d_new](https://user-images.githubusercontent.com/4588074/186404613-3826e9a7-85d5-415f-a394-bbd59c5e0f99.jpg) |


<details>
<summary>Day mode unchanged</summary>

| <img width="314" src="https://user-images.githubusercontent.com/4588074/186404772-84ce2697-e416-4ebc-bcbb-200f75fea970.jpg"> |
| --- |

No regression here :shipit:.
</details>

PR submission checklist:
